airflow/jobs/scheduler_job.py

```diff
@@ -1539,7 +1538,6 @@ def _execute_helper(self):
 
         # For the execute duration, parse and schedule DAGs
         while True:
-            self.log.debug("Starting Loop...")
```

Comment:
Why do you remove the debug logs?

Reply:
they slow it down, 'starting loop' on while true: seems meaningless